modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss

```diff
@@ -6,6 +6,11 @@
 	padding: 1rem;
 	position: inherit;
 
+	&-item-linked.treeview-link {
+		background-color: transparent;
+		color: rgba(107, 108, 126, 0.5);
```

Comment:
@alinedoleron @igor-franca Could you please use the hexadecimal code for this color? We don't usually use rgba.